api_proxy.yaml

```diff
@@ -463,7 +463,9 @@ definitions:
     required:
```

Comment:
can't we just reference the API definitions? it seems like they're the same type

Reply:
API definitions (`CreatedMessageWithContent`, `CreatedMessageWithoutContent`) have more fields like `time_to_live` and `fiscal_code`

Reply:
I don't see any problems passing everything to the app, @gunzip what do you think?

Reply:
Currently `fiscal_code` is not mandatory in `MessageWithContent` but it is in `CreatedMessageWithContent`. This would add this field to those already listed in the config file

Reply:
@lmorelli986 I don't understand exactly what you mean, what I'm saying is that, since the backend is just handing over to the app what's coming from the API, I don't see why the models should be different, makes sense?

Reply:
Loud and clear. And I did it. I was just pointing at a secondary collateral change, deriving from the decision.